server/app/controllers/dev/DatabaseSeedController.java

```diff
@@ -195,6 +289,33 @@ private QuestionDefinition insertRadioButtonQuestionDefinition() {
         .getResult();
   }
 
+  private QuestionDefinition insertStaticTextQuestionDefinition() {
+    return questionService
+        .create(
+            new StaticContentQuestionDefinition(
+                "static content",
+                Optional.empty(),
+                "description",
+                LocalizedStrings.of(
+                    Locale.US,
+                    "Hi I'm a block of static text. Welcome to this test program. It contains one"
+                        + " of every question type."),
+                LocalizedStrings.of(Locale.US, "")))
+        .getResult();
+  }
```

Comment:
NB: Is there any value adding an additional static question using the accordion formatting? https://docs.civiform.us/user-manual/civiform-admin-guide/manage-questions#question-types.

I'm not even sure if it's used in prod right now. It's pretty esoteric; you have to know how to set up the actual text.

Reply:
Ah cool, thanks for the pointer! I didn't know we supported formatting. I updated the static text.